charts/prometheus-rds-alerts/values.yaml

```
@@ -45,7 +45,7 @@ rules:
       description: '{{ $labels.dbidentifier }} has {{ printf "%.2g" $value }}% free disk space'
 
   RDSDiskSpacePrediction:
-    expr: predict_linear(rds_free_storage_bytes{}[30m], 3600 * 4) < 1
+    expr: predict_linear(min(rds_free_storage_bytes{}) by (aws_account_id, aws_region, dbidentifier)[30m:], 3600 * 4) < 1
```

```suggestion
    expr: predict_linear(min by (aws_account_id, aws_region, dbidentifier) (rds_free_storage_bytes{})[30m:], 3600 * 4) < 1
```

To maintain the same idea

---

Thanks, I applied the change